source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc

```diff
@@ -13,9 +13,12 @@ Ansible is used to initiate and manage the disaster recovery failover and failba
 ** Networks with the same general connectivity as the primary site.
 * Replicated storage. See <<storage_considerations_active-passive>> for more information.
 +
-NOTE: The replicated storage that contains virtual machines and templates must not be attached to the secondary site.
+[NOTE]
+====
+The replicated storage that contains virtual machines and templates must not be attached to the secondary site.
 * The `oVirt.disaster-recovery` package must be installed on the highly available Red Hat Ansible Engine machine that will automate the failover and failback.
```

Comment:
You made this and next sentences as part of the above comment...
Previously they were separate bullets.

Reply:
Good catch! Fixed.